persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/message/MessageDAOImpl.kt

```diff
@@ -33,8 +34,20 @@ class MessageDAOImpl(private val queries: MessagesQueries) : MessageDAO {
 
     override suspend fun deleteAllMessages() = queries.deleteAllMessages()
 
-    override suspend fun insertMessage(message: MessageEntity) = queries.transaction { insertInDB(message) }
+    override suspend fun insertMessage(message: MessageEntity, selfUserId: UserIDEntity, shouldUpdateConversationOrder: Boolean) {
```

Comment:
Thinking maybe we shouldn't move this business logic inside the persistence module but instead do something like this:

```
insertMessage(message: MessageEntity,  
              updateModifiedDate: Boolean = false, 
              updateReadDate: Boolean = false, 
              updateNotificationDate: Boolean: false) { ... }
```

And keep the logic in the logic module.

Reply:
Good point, will do that.